src/settings/settings_manager.cpp

```diff
@@ -0,0 +1,151 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// settings_manager.cpp
+//
+// Identification: src/settings/settings_manager.cpp
+//
+// Copyright (c) 2015-2017, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+
```

Comment:
The script formatter.py generates two empty lines automatically after the header information. Maybe we can reduce it by one?